example_configs/kimai.yaml

```diff
@@ -0,0 +1,36 @@
+# See https://www.kimai.org/documentation/local-yaml.html
+# this file should be renamed local.yaml
+kimai:
+  ldap:
+    activate: true
+    connection:
+      host: lldap
+      port: 3890
+      useSsl: false
+      useStartTls: false
+      username: uid=admin,ou=people,dc=example,dc=com
+      password: <PASSWORD_HERE>
+      accountFilterFormat: (&(objectClass=person)(uid=%s))
+      bindRequiresDn: true
+      optReferrals: false
+
+    user:
+      baseDn: ou=people, dc=example, dc=com
+      usernameAttribute: uid
+      filter: (&(objectClass=person)(memberof=cn=kimai,ou=groups,dc=example,dc=com))
```

Comment:
Add a top level note that you need to create the "kimai" group and add users to it

Reply:
Would moving the group filtering condition to a comment do the trick? Kimai doesn't require a group for LDAP to work.

Reply:
Yeah, you can move it to a comment, but still add a sentence (in the same comment) about the need to create the group.